xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -1192,6 +1192,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether doors are automatic.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="DoorControlElementHeight" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Indicates door control element height. This could be e.g. a door handle or a door button.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Indicates door control element height in meter. This could be e.g. a door handle or a door button.</xsd:documentation>
```